ticdc/ticdc-open-protocol.md

```diff
@@ -270,6 +270,13 @@ COMMIT;
 14. [partition=1] [key="{\"ts\":415508881038376963,\"t\":3}"] [value=]
 ```
 
+## Protocol parsing for consumers
+
+Currently, TiCDC does not provide the standard implementation of TiCDC Open Protocol parsing but demonstrations of parsing for the Golang version and Java version. You can refer to the data format provided in this document and the following demonstrations to implement the protocol parsing for consumers.
```

Comment:
```suggestion
TiCDC does not yet provide the standard implementation of parsing TiCDC Open Protocol for consumers. Currently it only provides the Golang and Java version of protocol parsing for demonstration. You can refer to the data format provided in this document and the following demonstrations to implement the protocol parsing for consumers.
```

Reply:
What about
Currently, TiCDC does not provide the standard parsing library for TiCDC Open Protocol, but Golang version and Java version demonstrations are provided.

Reply:
```suggestion
Currently, TiCDC does not provide the standard parsing library for TiCDC Open Protocol, but the Golang version and Java version of parsing demonstrations are provided. You can refer to the data format provided in this document and the following demonstrations to implement the protocol parsing for consumers.
```
LGTM @TomShawn @amyangfei

Reply:
Suggestion applied.